src/callbacks.rs

```diff
@@ -140,10 +107,44 @@ impl<'a> CompilerCalls<'a> for MiraiCallbacks {
 /// interpretation of all of the functions that will end up in the compiler output.
 fn after_analysis(state: &mut driver::CompileState) {
     let tcx = state.tcx.unwrap();
+    let crate_name: &str = state.crate_name.unwrap();
+    let summary_store_path: String = String::from(".summary_store.rocksdb");
```

Comment:
Is there a risk of concurrent tests clobbering each other because this path is always the same?

Reply:
There is, hence the removal of the other tests. As part of dealing with issue #10 I'll address this by adding a parameter that can create a unique temporary location for the DB.